lib/ansiblelint/utils.py

```diff
@@ -416,8 +416,14 @@ def normalize_task_v2(task: dict) -> dict:  # noqa: C901
 
     sanitized_task = _sanitize_task(task)
     mod_arg_parser = ModuleArgsParser(sanitized_task)
+    ansible_version = cli.CLI.version_info()
```

Comment:
This is no longer needed because we already require 2.9 or newer and we check for it.

I still think that skip_action_validation should be something that user can enable or disable using config file. Maybe disable them with.:
```
features:
  action-validation: false  # default true
```